eaw-functions-emmyluadoc.lua

```diff
@@ -0,0 +1,81 @@
+--######################################################################################
+--############################## Finder Functions ######################################
+--######################################################################################
+---@public
+---@param type_name string A type name as specified in the XML code
```

Comment:
Maybe we should note that as param it's case insensitive - but `GameObjectType` names returned from LUA functions are always `SCREAMING_SNAKE_CASE` thus it's probably goot practice to also use SSC for inputs, just to be on the safe side.